### DISCUSSION TITLE
2 touch screens

### DISCUSSION THREAD

**Discussion by Sam11100Narb** [GrandOrgue/grandorgue#1448] [Help]
Hello, 

Is it possible to use 2 touch screens to use stops on left and right panels, on each side of the keyboards ? I want to modify a GEM Concerto Organ.
Many Thanks

**Comment by ClarkeBrunt** [comment]
Only if the sample set provides additional panels showing the right and left stop jambs. The Friesach organ from Piotr Grabowski provides such panels - I don't know how many other sets might do. Or if you're just asking whether GO can use two touchscreens at all, then yes, though I've only got one myself.

Unless maybe you can do something with the main panel spanning both screens... Of course you could edit the ODF to provide extra panels, but not trivial.
